tests/rag_answers/custom_deepeval/metrics/context_relevancy/test_schemas.py

```diff
@@ -0,0 +1,81 @@
+from govuk_chat_evaluation.rag_answers.custom_deepeval.metrics.context_relevancy.schema import (
+    Truth,
+    TruthCollection,
+    Verdict,
+    VerdictCollection,
+)
+
+
+class TestTruthCollection:
+    def test_extracted_truths(self):
+        truth1 = Truth(context="ctx1", facts=["fact1", "fact2"])
+        truth2 = Truth(context="ctx2", facts=["fact3"])
+        truth_collection = TruthCollection(truths=[truth1, truth2])
+
+        facts = truth_collection.extracted_truths()
+        assert facts == [
+            {"context": "ctx1", "truths": ["fact1", "fact2"]},
+            {"context": "ctx2", "truths": ["fact3"]},
+        ]
+
+    def test_extracted_truths_empty(self):
+        truth_collection = TruthCollection(truths=[])
+
+        facts = truth_collection.extracted_truths()
+        assert facts == []
+
+    def test_extracted_truths_with_empty_truths(self):
+        truth1 = Truth(context="ctx1", facts=[])
+        truth2 = Truth(context="ctx2", facts=["fact1"])
+        truth_collection = TruthCollection(truths=[truth1, truth2])
+
+        facts = truth_collection.extracted_truths()
+        assert facts == [{"context": "ctx2", "truths": ["fact1"]}]
+
+
+class TestVerdictCollection:
```

Comment:
Minor: might be easier to read if these are ordered closer to the files organisation (so VerdictCollection is above TruthCollection as per the source file)